tests/installation/module_registration/verify_module_registration.pm

```diff
@@ -38,8 +39,12 @@ sub run {
         }
         else {
             foreach (@diff) {
-                # Hack prefix string "BUG#0#BUG" for workaround CI check of record_soft_failure
-                record_soft_failure("BUG#0#BUG:" . $softfail_modules_data{$_});
+                select_console 'install-shell';
+                my $error_msg = 'no sle-module-certifications in 15sp6';
+                die $error_msg if ($self->is_sles_in_rc_phase() || $self->is_sles_in_gm_phase());
```

Comment:
could you please create a function for this OR condition called "is_sles_in_rc_or_gm_phase" and replace it here and in `tests/installation/release_notes_from_url.pm`.

Reply:
OK